.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
 - docker run -d -p 9000:9000 -e "MINIO_ACCESS_KEY=AKIAIOSFODNN7EXAMPLE" -e "MINIO_SECRET_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY" minio/minio server /data
 - sudo apt-get -qq update
 - sudo apt-get install wget openjdk-7-jre redis-server imagemagick fontconfig libfontconfig
-- sudo apt-get install ffmpeg gstreamer0.10-ffmpeg ffmpegthumbnailer -y
+- sudo apt-get install ffmpegthumbnailer -y
```

Comment:
we don't need `ffmpeg` anymore ?

Reply:
Yes Karim just need ffmpegthumbnailer only

Reply:
oh because `ffmpegthumbnailer` already depends on `ffmpeg` :+1: